composer.json

```diff
@@ -20,14 +20,14 @@
   "require": {
     "php": "^8.0",
     "guzzlehttp/guzzle": "^7.4",
-    "laravel/framework": "^7.0|^8.0|^9.0"
+    "laravel/framework": "^8.0|^9.0|^10.0"
```

Comment:
Can we maintain Laravel 7 compatibility?

Reply:
```suggestion
    "laravel/framework": "^7.0|^8.0|^9.0|^10.0"
```